src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let cost = days * 40;
+
+  if (days >= 7) {
+    cost -= 50;
+  } else if (days >= 3) {
```

Comment:
Consider using a constant for the discount for rentals of 3 days or more (20) for better readability and easier maintenance. 🎁